UnitsNet/UnitDefinitions/Torque.json

```diff
@@ -52,12 +52,12 @@
       "PluralName": "PoundForceInches",
       "FromUnitToBaseFunc": "x*0.1129848388",
       "FromBaseToUnitFunc": "x*8.85074502580075",
-      "Prefixes": [ "Kilo" ],
+      "Prefixes": [ "Kilo", "Mega" ],
       "Localization": [
         {
           "Culture": "en-US",
           "Abbreviations": [ "lbf·in" ],
-          "AbbreviationsWithPrefixes": [ "kipf·in" ]
+          "AbbreviationsWithPrefixes": [ "kipf·in", "Mlbf·in"]
```

Comment:
It seems to me that `kipf` or `kip` is a special case of widely used abbreviations although it "should" have been named `klbf·in` for consistency. Unfortunately the current code generator doesn't support multiple units per prefix, so we have to choose one and I won't change it now to avoid breaking anything. I'm creating a new issue to address the need to support multiple abbreviations for prefixed units so we can add it later. See #344

You don't have to take any action in this PR, this is just an observation on my part.